coding-the-legislation/10_basic_example.md

```diff
@@ -61,6 +61,54 @@ To make sure that the formula you have just written works the way you expect, yo
 
 You can check the [YAML tests documentation](writing_yaml_tests.md) to learn more about how to write YAML tests, and how to run them.
 
+## Example with Enum variable
```

Comment:
This should probably not be in “Basic example: coding a simple formula”. We are not coding a formula but an input variable here. What about moving it as an “Advanced” example of section 2.2 “Introducing an input variable”? 🙂